src/api/extensions/gls-parcelshop-dk/index.js

```diff
@@ -22,7 +22,23 @@ module.exports = ({config, db}) => {
         returnData.droppoints = []
 
         // pass the request to elasticsearch
-        const url = 'http://' + config.elasticsearch.host + ':' + config.elasticsearch.port + '/gls_parcelshop_dk/droppoint/_search';
+        let port = config.elasticsearch.port
+        if (!port || port.length < 2) {
+            port = ''
+        } else {
+            port = `:${port}`
+        }
+        let url = `${config.elasticsearch.protocol}://${config.elasticsearch.host}${port}/gls_parcelshop_dk/droppoint/_search`
```

Comment:
This is something also repeated few times in the code and should be separated to the shared helper function

Reply:
Also it should be const instead of let